avro_schema/frontend.lua

```diff
@@ -978,9 +942,48 @@ function debug_print_indent(msg)
     print(msg)
 end
 
+--
+-- Compiler generates code from the IR, which is more generic than
+-- a bare schema. IR defines a 'from' schema, a 'to' schema and a mapping.
```

Comment:
Generated code means lua code? It worth to state it here.

Don’t get the idea about being 'more generic' then a 'bare schema', can you expand it?

Reply:
Rewritten.